File: backend/app/Models/Card.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Concerns\HasUuids;
use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Card extends Model
{
    use HasFactory, HasUuids;

    protected $fillable = [
        'deck_id',
        'front',
        'back',
        'example_sentence',
        'word_type',
        'image_url',
        'audio_url',
    ];
    
    public function deck()
    {
        return $this->belongsTo(Deck::class);
    }
}

File: shared/src/commonMain/kotlin/com/vocapop/domain/models/Card.kt
package com.vocapop.domain.models

import kotlinx.serialization.Serializable

@Serializable
data class Card(
    val id: String,
    val deckId: String,
    val front: String,
    val back: String,
    val exampleSentence: String? = null,
    val wordType: String? = null, // Noun, Verb, etc.
    val imageUrl: String? = null,
    val audioUrl: String? = null
)
